package.json

```diff
@@ -23,23 +25,30 @@
   },
   "dependencies": {
     "moment": "^2.10.6",
-    "react": "^0.13.3",
-    "react-bootstrap": "^0.26.1",
-    "react-datepicker": "^0.13.0",
-    "react-time-select": "^1.0.2"
+    "react": "^0.14.2",
```

Comment:
After speaking with @shackpank, I'm happy for module versions to update within minor versions. The thinking is that this is client side. If we were to lock down version numbers, then the client could potentially end up with numerous different versions of the same module.